.github/workflows/deployment.production.yml

```diff
@@ -0,0 +1,115 @@
+name: Deployment production
+
+on:
+  release:
+    types: [published]
+
```

Comment:
Let's add the NX_CLOUD_ACCESS_TOKEN variable to leverage Nx Cloud distributed cache

Reply:
add the token as an environments variable